### PR TITLE
test: fix Symbol error in fixture

### PR DESCRIPTION
**Pull request by nolanlawson**
## Details

I didn't notice in #4730 that we introduced an `error.txt` caused by an attempt to convert a `Symbol` to a string. Since using `Symbols` in this case is a super-rare edge case, and because it reduces the value of the test to just have an error message, I say we remove the `Symbol`.

## Does this pull request introduce a breaking change?

-   😮‍💨 No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

-   🤞 No, it does not introduce an observable change.

### CODE REVIEW

File: packages/@lwc/engine-server/src/__tests__/fixtures/known-boolean-attributes/default-def-html-attributes/dynamic/error.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-Cannot convert a Symbol value to a number
```
Comment (wjhsf) [inline packages/@lwc/engine-server/src/__tests__/fixtures/known-boolean-attributes/default-def-html-attributes/dynamic/error.txt:1]: Wait why was it trying to convert to a number and not a string?

**Review by wjhsf — APPROVED** [review]

File: packages/@lwc/engine-server/src/__tests__/fixtures/known-boolean-attributes/default-def-html-attributes/dynamic/error.txt (hunk 1 above)
Comment (nolanlawson) [reply packages/@lwc/engine-server/src/__tests__/fixtures/known-boolean-attributes/default-def-html-attributes/dynamic/error.txt:1]: I'm assuming because of `tabindex`.
